packages/@dcl/sdk-commands/src/logic/bundle.ts

```diff
@@ -74,8 +74,8 @@ export async function bundleProject(components: BundleComponents, options: Compi
     const entryPointCode = options.customEntryPoint
       ? `export * from '${filePath}'`
```

Comment:
```suggestion
      ? `export * from '@dcl/sdk'; export * from '${filePath}'`
```

Reply:
For `customEntryPoint` you take the responsibility of using `@dcl/sdk` or a modified one.